pkg/ceph/client/pool.go

```diff
@@ -127,15 +127,22 @@ func CreatePool(context *clusterd.Context, clusterName string, newPool CephStora
 		}
 	}
 
+	// ensure that the newly created pool gets an application tag (just reuse the pool name)
+	args = []string{"osd", "pool", "application", "enable", newPool.Name, newPool.Name}
```

Comment:
Those docs indicate to me that `rbd` is set as the app name when the rbd tool is being used and when you know that the pool is going to be used for rbd.  We have no guarantee of that here because this is just a general create new pool function.  The pool could be used for anything.

See https://github.com/rook/rook/issues/889#issuecomment-321443969, where it's explained that for pools created for custom applications, the name can be anything.  It's the hope that the pool name describes what it's going to be used for, and therefore reusing it for the pool's app name sounds like an OK approach.